docs/application/web/guides/multimedia/media-controller.md

```diff
@@ -521,6 +531,166 @@ To manage the media controller playlist in your application, you must handle req
    mcServerInfo.removePlaylistUpdatedListener(watcherId);
    ```
 
+## Server abilities
+
+Multiple server's abilities can be set to give information to the client about supported features. Abilities can be divided into two groups:
+1. Media controller simple abilities - each ability is described by a single [MediaControllerAbilitySupport](../../api/latest/device_api/mobile/tizen/mediacontroller.html#MediaControllerAbilitySupport) value and is not a part of a complex ability structure
+
+    - PLAYBACK_POSITION - Ability to change playback position.
+    - SHUFFLE - Ability to change shuffle mode.
+    - REPEAT - Ability to change repeat state.
+    - PLAYLIST - Ability to add/change/remove playlists.
+    - CLIENT_CUSTOM - Ability to receive custom commands from media controller clients.
+    - SEARCH - Ability to receive search requests from media controller clients.
+    - SUBTITLES - Ability to receive requests for subtitles mode change from media controller clients.
+    - MODE_360 - Ability to receive requests for spherical (360&deg;) mode change from media controller clients.
+
+2. Media controller complex abilities - each ability consists of several types.
+
+    - playback - Abilities of server's playback actions.
+    - displayRotation - Abilities of setting display orientations.
+    - displayMode - Abilities of setting display modes.
+
+To set abilities on server side you can use `saveAbilities()` method.
+
+```
+server.updatePlaybackState("PLAY");
+server.abilities.playback.next = "YES";
+server.abilities.playback.prev = "YES";
+server.abilities.playback.rewind = "NO";
+server.abilities.playback.forward = "NO";
+
+server.abilities.playback.saveAbilities();
+```
+
+Keep in mind that using `saveAbilities()` is required to save changes of playback abilities into database, otherwise changes will have no effect on the device and clients will not be notified about an update.
+
+To get abilities of the server on the client side:
+
+```
+info = client.getLatestServerInfo();
+console.log("After save:");
+console.log("ability NEXT: " + info.abilities.playback.next);
+console.log("ability PREV: " + info.abilities.playback.prev);
+console.log("ability REWIND: " + info.abilities.playback.rewind);
+console.log("ability FORWARD: " + info.abilities.playback.forward);
+```
+
+You can also monitor server's changes of its abilities by using ` addAbilityChangeListener` method on client side.
```

Comment:
' addAbilityChangeListener' -- why space at the beginning?

Reply:
Nobody knows...